spec/frameSpec.js

```diff
@@ -0,0 +1,38 @@
+describe("Frame", function() {
+
+  var frame;
+
+  beforeEach(function() {
+    frame = new Frame();
+  });
+
+  it("can have two rolls", function(){
```

Comment:
The description of the test is not quite accurate - you are actually testing that it can add the scores of two rolls